plugins/UsersManager/templates/_userInviteEmail.twig

```diff
@@ -1,8 +1,18 @@
+{% set inviteLinks = postEvent('UsersManager.setInviteLinks', login) %}
+{% set notes = postEvent('UsersManager.setInviteNotes', login)|default(notes) %}
```

Comment:
I would probably call the methods as `getInviteLinks` and `getInviteNotes` as they are obtaining that content via the event, not setting it.